document-store/src/main/java/org/hypertrace/core/documentstore/mongo/MongoDatastore.java

```diff
@@ -91,6 +93,15 @@ public DocStoreMetricProvider getDocStoreMetricProvider() {
     return docStoreMetricProvider;
   }
 
+  @Override
+  public void close() {
+    try {
+      client.close();
```

Comment:
can client be null here by any chance?

Reply:
Nope. The client is initialized in the constructor and hence, cannot be null.